cmd/lassie/lassie.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"log"
+	"os"
+	"time"
+
+	"github.com/urfave/cli/v2"
+)
+
+func main() { os.Exit(main1()) }
+
+func main1() int {
+	app := &cli.App{
+		Name:  "lassie",
+		Usage: "Utility for retrieving content from the Filecoin network",
+		Commands: []*cli.Command{
+			{
```

Comment:
it might make sense to put fetch config under fetch.go, but not super important for now (so this would read:
```golang
app := &cli.App{
   // ...
   Commands: []*cli.Command{
      fetchCmd,
   },
}
```

while fetch.go would have:

```golang
fetchCmd = &cli.Command{
   // fetch config args
}
```
)